tools/rosbag_storage/src/bag_player.cpp

```diff
@@ -63,7 +63,6 @@ void BagPlayer::start_play() {
 }
 
 void BagPlayer::unregister_callback(const std::string &topic) {
-    delete cbs_[topic];
```

Comment:
This means it will now not be deleted until the whole object goes out of scope. Perhaps there should still be a `cbs_.erase(topic)` or something here?

Reply:
Of course, that's in the next line. I didn't remove it.

Reply:
Oh lol, so I see.